config.nims

```diff
@@ -113,6 +113,9 @@ switch("define", "libp2p_pki_schemes=secp256k1")
 # we can't use it. And codex.cfg doesn't work
 switch("define", "chronicles_sinks=textlines[dynamic],json[dynamic],textlines[dynamic]")
 
+# Workaround for assembler incompatibility between constantine and secp256k1
+switch("define", "use_asm_syntax_intel=false")
```

Comment:
Where is this flag coming from, I cant seem to find it.

Reply:
It's mentioned in the README, due to an issue with Clang, LTO, AT&T assembly syntax and constant propagation: https://github.com/mratsim/constantine/blob/5f7ba18f2ed351260015397c9eae079a6decaee1/README.md?plain=1#L159


https://github.com/mratsim/constantine/blob/5f7ba18f2ed351260015397c9eae079a6decaee1/constantine/platforms/isa/macro_assembler_x86.nim#L11


https://github.com/mratsim/constantine/blob/5f7ba18f2ed351260015397c9eae079a6decaee1/constantine/platforms/config.nim#L31

Reply:
Thanks! (yay for nim's case insensitivity `UseAsmSyntaxIntel`)